### PR TITLE
Add french translations

### PR DESCRIPTION
**Pull request by Molkobain**

### CODE REVIEW

**Comment by AHalfWildSheep** [comment]
Hi Molkobain, thanks for the contribution! 😊
I corrected a dictionary entry key and added a new one with 9a9d548b, would you mind adding them to your PR?

Regards

File: fr.dict.ahws-dashlet-stats.php [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+	'UI:DashletStats:Prop:UnitPosition' => 'Position de l\'unité',
+	'UI:DashletStats:Prop:UnitPosition:Before' => 'Avant',
+	'UI:DashletStats:Prop:UnitPosition:After' => 'Après',
+	'UI:DashletStats:Prop:Function:PercentageQuery' => 'Requête en pourcentage',
+
+
+	'UI:DashletStatsCompare:Label' => 'Dashlet de stats. comparatives',
+	'UI:DashletStatsCompare:Description' => 'Dashlet badge pour comparer deux fonctions d\'aggrégation',
+	'UI:DashletStatsCompare:Prop:CompareQuery' => 'Requête de comparaison',
+	'UI:DashletStatsCompare:Prop:CompareUnit' => 'Unité de comparaison del\'écart',
```
Comment (steffunky) [inline fr.dict.ahws-dashlet-stats.php]: Missing space between "de" et "l'écart"

Comment (Molkobain) [reply fr.dict.ahws-dashlet-stats.php]: Done !

**Comment by Molkobain** [comment]
> Hi Molkobain, thanks for the contribution! 😊
> I corrected a dictionary entry key and added a new one with [9a9d548](https://github.com/AHalfWildSheep/itop-dashlet-stats/commit/9a9d548b0b8aeb3cd5533e4897bc10c65246670b), would you mind adding them to your PR?
> 
> Regards

Sure, I update my branch soon and let know

**Comment by Molkobain** [comment]
I'm closing this PR as I messed up. Modifications were on the master branch, so the repo is now out of sync. Re-doing it on a dedicated branch.

**Pull request closed without merging**